sdks/python/apache_beam/runners/portability/spark_runner_test.py

```diff
@@ -42,6 +42,8 @@
   parser.add_argument('--environment_type', default='docker',
                       help='Environment type. docker or process')
   parser.add_argument('--environment_config', help='Environment config.')
+  parser.add_argument('--environment_cache_millis', default=0,
```

Comment:
Instead of introducing a default here, shall we rely on the system default and do the null check before adding `portable_options.environment_cache_millis` in line 96.

Reply:
Updated